### DISCUSSION TITLE
Additional columns in reverse object relation

### DISCUSSION THREAD

**Discussion by betterapp** [pimcore/pimcore#11130] [Q&A / Support]
Hi.

We have two classes. Product and Category.
Product have relation to category. From category class side there is a reverse relation to product class.

But with reverse relation I can not define columns which I want to show from category object side.
Is there a way to do this ?

**Comment by NiklasBr** [comment]
Are you using [Advanced relation types](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Relation_Types.html#page_Advanced-Many-To-One-Object-Relation)?

**Reply by betterapp** [reply]
No.

**Reply by betterapp** [reply]
but it does not matter it is not related with it

**Reply by NiklasBr** [reply]
Aha, I thought you maybe wanted to add additional columns, but maybe you want to show all available columns? In that case, make sure they are selected in the Class Definition:

![bild](https://user-images.githubusercontent.com/279826/148222782-3ecb62fd-131f-4923-b848-898289335afc.png)

**Comment by betterapp** [comment]
The only columns I see in Reverse relations are:
![image](https://user-images.githubusercontent.com/7449124/148211955-6ecaf521-b709-4866-bbe3-4e6f379cdadd.png)

**Reply by cancan101** [reply]
Did you find a solution?

**Reply by betterapp** [reply]
I did not have a time to think about it ;)
